app/components/layout/Navigation/MyBuilds/index.js

```diff
@@ -105,14 +111,16 @@ class MyBuilds extends React.Component {
   renderBadge() {
     const buildsCount = this.state.runningBuildsCount + this.state.scheduledBuildsCount;
 
-    // Only render the badge if we've actually got a number to show
-    if (buildsCount) {
-      return (
-        <Badge className={classNames("hover-lime-child", { "bg-lime": this.state.isDropdownVisible })}>
-          {buildsCount}
-        </Badge>
-      );
+    // Render nothing (an empty span) if we've not got a number to show
+    if (!buildsCount) {
+      return <span />;
```

Comment:
I think we can return `null`? (Or do we need to return a `<span>` for animations to work?)

Reply:
No, we need to return something that actually renders because otherwise the `CSSTransition` complains that it has no children. I honestly consider it a bug, but I think it’s a side effect of the way we’re using it.